src/ks-app/src/app/containers/buttons/button-groups.component.html

```diff
@@ -0,0 +1,295 @@
+            <button class="btn">3</button>
+            <button class="btn">4</button>
+        </div>
+    </div>
+    <div class="col">
+        <h4>Warning Outline</h4>
+        <div class="btn-group btn-sm btn-outline-warning">
+            <button class="btn">1</button>
+            <button class="btn">2</button>
+            <button class="btn">3</button>
+            <button class="btn">4</button>
+        </div>
+    </div>
+    <div class="col">
+        <h4>Disabled Outline</h4>
+        <div class="btn-group btn-sm btn-outline-warning">
+            <button disabled class="btn">1</button>
+            <button disabled class="btn">2</button>
+            <button disabled class="btn">3</button>
+            <button disabled class="btn">4</button>
+        </div>
+    </div>
+</div>
+
+<h3>Button Group Links</h3>
+<div class="content-wrapper">
+    <div class="col">
+        <div class="btn-group btn-link">
+            <button class="btn">1</button>
+            <button class="btn">2</button>
+            <button class="btn">3</button>
+            <button class="btn">4</button>
+        </div>
+    </div>
+</div>
+
+<h3>Inverse Button Group</h3>
+<div class="content-wrapper">
+    <div class="col inverse">
+        <div class="btn-group btn-inverse">
+            <button class="btn">1</button>
+            <button class="btn">2</button>
+            <button class="btn">3</button>
+            <button class="btn">4</button>
+        </div>
+    </div>
+</div>
+
+<h3>Button Group w/ Disabled</h3>
+<div class="content-wrapper">
+    <div class="col">
+        <clr-button-group>
+            <clr-button name="button1" id="test">1</clr-button>
+            <clr-button type="button">2</clr-button>
+            <clr-button disabled>3</clr-button>
+            <clr-button disabled="false">4</clr-button>
+            <clr-button disabled="123">5</clr-button>
+            <clr-button>6</clr-button>
+        </clr-button-group>
+    </div>
+</div>
+
+<h3>Button Group w/ Overflow</h3>
+<div class="content-wrapper">
+    <div class="col">
+        <clr-button-group class="btn-primary">
+            <clr-button>1</clr-button>
+            <clr-button>2</clr-button>
+            <clr-button>3</clr-button>
+            <clr-button>4</clr-button>
+            <clr-button [clrInMenu]="true">5</clr-button>
+            <clr-button [clrInMenu]="true">6</clr-button>
+            <clr-button [clrInMenu]="true">7</clr-button>
+        </clr-button-group>
+    </div>
+</div>
+
+<h3>Mixed Button Group</h3>
+<div class="content-wrapper">
+    <div class="col">
+        <clr-button-group class="btn-primary">
+            <clr-button>1</clr-button>
+            <clr-button>2</clr-button>
+            <clr-button class="btn btn-success">3</clr-button>
+            <clr-button class="btn btn-danger">4</clr-button>
+            <clr-button>5</clr-button>
+        </clr-button-group>
+    </div>
+</div>
+
+<h3>Icon Button Groups</h3>
```

Comment:
We are missing the new Icon Buttons here. After the docs are published tomorrow, this section might need to be updated.